service/docs/source/active_session.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://geopm.github.io/active_session.schema.json",
+    "title": "ActiveSession",
+    "type": "object",
+    "properties": {
+      "client_pid": {
+        "type": "number"
+      },
+      "reference_count": {
+        "type": "integer",
+        "minimum": 0
+      },
+      "signals": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "controls": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "watch_id": {
+        "type": "number"
+      },
+      "batch_server": {
+        "type": "number"
+      },

```

Comment:
Same comment here as with the `client_pid`. I think we can change these all to `integer`.